de-lib/src/main/java/org/iplantc/de/client/models/tool/ToolAutoBeanFactory.java

```diff
@@ -33,4 +34,20 @@
 
     AutoBean<SimpleServiceError> simpleServiceError();
 
+    /**
+     * Build a tool with default values
+     *
+     * @return  a tool
+     */
+    default AutoBean<Tool> getDefaultTool() {
```

Comment:
I still think it might be a good idea for the `ToolAdminDetailsDialog` to take advantage of this new default method.